packages/terra-application-layout/translations/pt.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Are we still waiting on translations?

Reply:
Nah, what's here is from the translations folks. Might just be one of those words that are the same across languages? Anyone speak Portuguese?

Reply:
Translations are weird...
![screen shot 2018-03-23 at 11 05 31 am](https://user-images.githubusercontent.com/14099737/37840088-2989b8f6-2e8a-11e8-92e0-daaad9e7045d.png)